packages/sexp_decode/sexp_decode.0.2/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "A library to decode S-expression into structured data"
+description:
+  "A library of monadic combinators that help translating S-expressions (as provided by the Csexp library) into structured data"
+maintainer: "Benoît Montagu <benoit.montagu@inria.fr>"
+authors: "Benoît Montagu <benoit.montagu@inria.fr>"
+license: "LGPL-3.0-or-later"
+homepage: "https://gitlab.inria.fr/bmontagu/sexp_decode"
+bug-reports: "Benoît Montagu <benoit.montagu@inria.fr>"
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml" {>= "4.08"}
```
This is where let+ and let* where introduced (will solve the syntax error, operator expected failure)